client.go

```diff
@@ -128,7 +128,10 @@ type Client interface {
 	LSPGetDHCPv6Options(lsp string) (*DHCPOptions, error)
 	// Set options in LSP
 	LSPSetOptions(lsp string, options map[string]string) (*OvnCommand, error)
-
+	// Set dynamic addresses in LSP
+	LSPSetDynamicAddresses(lsp string, address string) (*OvnCommand, error)
+	// Get dynamic addresses from LSP
+	LSPGetDynamicAddresses(lsp string) (string, error)
```

Comment:
I think the "LSPGetDynamicAddresses" is not necessary. If you get the LSP struct with LSPGet(), you can access any member of it directly. Any benefit of adding this interface separately?

Reply:
`GetDynamicAddresses` is just a convenience function.

Reply:
Note that `GetDynamicAddresses` also folds `libovsdb.OvsSet` into a single string as expected by the nbdb spec.

Reply:
ok